### ISSUE TITLE
What is the ID associated with the pem key?

### ISSUE THREAD

**Issue by vitalbone** [awslabs/amplify-video#170]
When adding or updating the video category to allow for signed urls, I am a little confused as to what `ID` in the `What is the ID associated with the pem key?` is referring to.

Is it:
- The AWS Account ID?
- The Access Key ID?
- The RSA Key ID?

```sh
➜ amplify video update
? Choose what project you want to update? myvideoproject
? Select a system-provided encoding template, specify an already-created template name: HLS
? Is this a production enviroment? Yes
? Do you want to protect your content with signed urls? Yes
? Provide the location to the pem key you want CloudFront you want to sign urls with: /path/to/my/private-key.pem
? What is the ID associated with the pem key? <------
```

As always, thanks for the great lib @wizage.

**Comment by smp** [comment]
It's the CloudFront Access Key ID, created a pr to update: https://github.com/awslabs/amplify-video/pull/173

**Comment by vitalbone** [comment]
Thanks @smp!

**Comment by wizage** [comment]
Merged with #173. Pending release
